frontend/src/components/homepage/sections/HomepageBuilding.tsx

```diff
@@ -10,35 +10,37 @@ const BuildingContent = () => {
   const { paragraphs } = messages.Homepage.sections.building;
 
   return (
-    <GridContainer data-testid="homepage-building">
-      <Grid row gap="md" className="padding-6">
+    <GridContainer
+      data-testid="homepage-building"
+      className="padding-y-4 tablet-lg:padding-y-6"
+    >
+      <Grid row gap>
         <Grid
-          tablet={{
+          tabletLg={{
             col: true,
           }}
+          className="tablet-lg:padding-right-6"
         >
           <Image
             alt="events-img"
-            className="height-auto position-relative padding-right-6 padding-top-3"
+            className="height-auto position-relative margin-bottom-2 tablet-lg:margin-bottom-0"
             src={BuildingImage}
           />
```

Comment:
- Use a large tablet grid (image is too large for 2-column on smaller tablet)
- Space the image appropriately from the heading on various sizes (with the padding on the grid instead of image)